### ISSUE TITLE
A11y_AzureDataStudioAAE_Azure ADC(Preview)_Azure Configuration_ScreenReader: Screenreader does not announces 'Azure details' is expanded or collapsed while navigating with tab key.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#15728]
https://user-images.githubusercontent.com/85863999/127467658-30d3ff46-949f-4900-9121-5152eeb95022.mp4

### Actual:
While navigating with tab key & focus is on 'Azure details' , press enter for collapse and when user again press enter to expand, NVDA not announces 'Azure details' is expanded.
Similar issue is observed with JAWS.

Similar issue is present in below scenarios:
Azure Arc Data Controller(Preview)-Controller Configuration
Azure Arc Data Controller(Preview)-Review Your Configuration
PostgreSQl Hyperscale server groups-Provide azure enabled PostgreSQl Hyperscale server groups
Azure Sql Managed Instance- Provide azure sql managed instance parameters

### Expected:
While navigating with tab key & focus is on 'Azure details', press enter for collapse and when user again press enter to expand, NVDA should announces 'Azure details' is expanded.

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#STP;#WCAG1.3.1;#NVDA;#Jaws;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10
Screen reader: NVDA(2020.2), Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till "Connection" and tab to see more button
4. Tab to "New Deployment" and Azure ADC(Preview) and then tab select button
5. Deployment Prerequisite page will open and tab on next "Select from existing Kubernetes cluster" will open
6. Start NVDA
7. Tab till "Azure Details" and check how NVDA announces information 

### User Impact:
Screenreader user will not able to get complete information of 'Azure details'.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by Shivamd1** [comment]
![image](https://user-images.githubusercontent.com/60880975/150973644-e8f94cbc-049f-41ee-a7c9-c643baec6c5f.png)

**Comment by Shivamd1** [comment]
Closing not repro
